x/ibc/04-channel/keeper/timeout.go

```diff
@@ -196,3 +196,25 @@ func (k Keeper) TimeoutOnClose(
 
 	return packet, nil
 }
+
+// DeleteCommitmentTimeout deletes the commitment send from this chain after it verifies timeout
+func (k Keeper) DeleteCommitmentTimeout(ctx sdk.Context, packet exported.PacketI) error {
```

Comment:
the function name is not accurate as it also updates the channel if it's ordered. Consider moving out or updating the name + comment

Reply:
renamed to `TimeoueExecuted`